pdns/dnsdist-lua.cc

```diff
@@ -158,7 +158,7 @@ std::unordered_map<int, vector<boost::variant<string,double>>> getGenResponses(u
   vector<pair<int, DNSName>> rcounts;
   rcounts.reserve(counts.size());
   for(const auto& c : counts) 
-    rcounts.push_back(make_pair(c.second, c.first));
+    rcounts.push_back(make_pair(c.second, c.first.makeLowerCase()));
```

Comment:
Nope since it uses maps, which uses the default compare and less operators, which in turn lowercase everything themselves.